README.md

```diff
@@ -6,6 +6,11 @@
 
 **Factory interfaces for PSR-7 HTTP Message.**
 
+## Deprecation notice
+
+This package and these PSR-7 factory interfaces are deprecated. We have succeeded with our goal of moving these
+interfaces to PHP-FIG. The new [PSR-17](https://packagist.org/packages/psr/http-factory) package is backwards compatible
```

Comment:
I don't think it's true. The API might not break, but you won't get the same PSR-7 object.

Reply:
well, if its the same http message, it should be BC. if its not the same PSR-7 implementation, that should not matter, as code should not rely on a factory returning a specific implementation. or do you mean something else?